app/views/distributions/_distribution_item_total.html.erb

```diff
@@ -1,6 +1,7 @@
 <tr>
-  <td></td>
   <td><strong>Total:</strong></td>
-  <td><strong><%= item_value(@distribution.value_per_itemizable) %></strong></td>
+  <td></td>
+  <td class="numeric"><strong><%= item_value(@distribution.value_per_itemizable) %></strong></td>
+  <td></td>
   <td></td>
```

Comment:
I *think* you're ending up with an additional `<td>` tag here -- is that intentional? (1 deleted, 1 replaced, 2 added)

Reply:
.

Reply:
@armahillo yes, that is intentional. I didn't add extra `<td>`, it was already there, I moved them to make the `Total` align with others.

Reply:
Keith mentioned about this here: https://github.com/rubyforgood/diaper/pull/1263#issuecomment-541301422